workloads/files/workload-mastervertical-script-cm.yml

```diff
@@ -80,7 +80,8 @@ data:
       result_dir=${benchmark_results_dir}
     fi
     start_time=$(date +%s)
-    VIPERCONFIG=/tmp/mastervertical.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster concurrently with templates [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
+    git clone https://github.com/cloud-bulldozer/snafu; pip install -r snafu/requirements.txt;
+    VIPERCONFIG=/tmp/mastervertical.yaml python snafu/run_snafu.py -t cl --cl-output True | tee "${result_dir}/clusterloader.txt"
```

Comment:
Have you made sure the `mastervertical.yamls` is executed? When I previously ran with snafu, I had to `export VIPERCONFIG` instead of just setting a local shell var.

Reply:
> Have you made sure the `mastervertical.yamls` is executed? When I previously ran with snafu, I had to `export VIPERCONFIG` instead of just setting a local shell var.

I did a simple test to see if that's the case and looks like local shell var is enough:
```
[root@smicro-6029p-04 ~]# unset X
[root@smicro-6029p-04 ~]# printenv | grep X
XDG_SESSION_ID=2434
SELINUX_ROLE_REQUESTED=
SELINUX_USE_CURRENT_RANGE=
TMUX=/tmp/tmux-0/default,23441,6
XMODIFIERS=@im=ibus
TMUX_PANE=%19
SELINUX_LEVEL_REQUESTED=
XDG_RUNTIME_DIR=/run/user/0
[root@smicro-6029p-04 ~]# X=aakarsh python x.py 
XDG_SESSION_ID=2434
SELINUX_ROLE_REQUESTED=
SELINUX_USE_CURRENT_RANGE=
X=aakarsh
TMUX=/tmp/tmux-0/default,23441,6
XMODIFIERS=@im=ibus
TMUX_PANE=%19
SELINUX_LEVEL_REQUESTED=
XDG_RUNTIME_DIR=/run/user/0
[root@smicro-6029p-04 ~]# printenv | grep X
XDG_SESSION_ID=2434
SELINUX_ROLE_REQUESTED=
SELINUX_USE_CURRENT_RANGE=
TMUX=/tmp/tmux-0/default,23441,6
XMODIFIERS=@im=ibus
TMUX_PANE=%19
SELINUX_LEVEL_REQUESTED=
XDG_RUNTIME_DIR=/run/user/0
[root@smicro-6029p-04 ~]# cat x.py 
import subprocess

subprocess.check_call("printenv | grep X", shell=True)
```